src/libraries/System.Runtime.InteropServices/tests/LibraryImportGenerator.UnitTests/Diagnostics.cs

```diff
@@ -295,7 +295,7 @@ partial class Test
         }
 
         [Fact]
-        [OuterLoop("Uses the network for downlevel ref packs")]
+        //[OuterLoop("Uses the network for downlevel ref packs")]
```

Comment:
Is this intentional?

Reply:
No that was just needed to be able to run the test in VS.

```suggestion
        [OuterLoop("Uses the network for downlevel ref packs")]
```